CHANGELOG.md

```diff
@@ -0,0 +1,27 @@
+# Changelog
+
+All notable changes to the `XgridInc/xc3` repository will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [1.0.0] - 2023-05-11
+
+### Added
+- Initial release of the `XgridInc/xc3` repository.
+- `cloud_custodian_policies` folder which has the yaml for all the cloud custodian policies
```

Comment:
- `cloud_custodian_policies` folder containing the YAML file for all the cloud custodian policies

Reply:
Resolved in [bea9bfe](https://github.com/XgridInc/xc3/pull/8/commits/bea9bfefc5c3042b6a9437670d0ac2db8d12b89d)